scripts/hostapd/nat.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Forward all IPv4 traffic from the internal network to the eth0 device and mask with the eth0 external IP address
+iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
+
+# Forward all IPv6 traffic from the internal network to the tun0 device and mask with the tun0 external IP address
+ip6tables -t nat -A POSTROUTING -o tun0 -j MASQUERADE
```

Comment:
>cjd | benhylau: what you have will do what you want because already tun0 will only have a route on it which routes fc00::/8
>madafoo | benhylau, traffic outoing to which interface is chose by the route table, so u do not need do anything at all.

No change needed